apps/backoffice-v2/src/pages/Entity/components/Case/hooks/usePendingRevisionEvents/usePendingRevisionEvents.tsx

```diff
@@ -0,0 +1,60 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { useCallback, useMemo } from 'react';
+import { calculateAllWorkflowEvents } from '@/pages/Entity/components/Case/hooks/usePendingRevisionEvents/utils/calculate-pending-workflow-events';
+import { CommonWorkflowEvent } from '@ballerine/common';
+import { checkIsKybExampleVariant } from '@/lib/blocks/variants/variant-checkers';
+import { useRevisionCaseMutation } from '@/domains/workflows/hooks/mutations/useRevisionCaseMutation/useRevisionCaseMutation';
+import { IPendingEvent } from './interfaces';
+
+function composeUniqueWorkflowEvents(
+  acc: Record<string, IPendingEvent>,
+  pendingWorkflowEvent: IPendingEvent,
+) {
+  acc[`${pendingWorkflowEvent?.workflowId}-${pendingWorkflowEvent?.pendingEvent}`] =
+    pendingWorkflowEvent;
+
+  return acc;
+}
+
+function isRevisionEvent(pendingWorkflowEvent: IPendingEvent) {
```

Comment:
I think the problem here is the name of the function itself. It's better that there are several conditions and that they may evolve over time so the implementation is not reflected directly in the naming but `isRevisionEvent` gives the wrong impression.

Reply:
i updated it to isPendingEventIsRevision, but I dont think it's still proper